pkgs/tools/filesystems/ioztat/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3 }:
```

Comment:
Small formatting nitpick: keep `}:` on a separate line.
You can try out `nixpkgs-fmt`.